spec/client_spec.js

```diff
@@ -249,22 +267,25 @@ describe('Client', function() {
           source: subject._source,
           data: { id: requestsCount - 1, result: {a: 'b'} }
         });
-        ++requestsCount;
       });
     });
 
     describe('#set', function() {
       it('returns a promise', function() {
-        expect(subject.set('ticket.subject', 'value')).to.be.a.promise;
+        var promise = subject.set('ticket.subject', 'value');
+
+        promise.catch(function() {});
```

Comment:
yeah i tried that, but then promise refers to the wrong promise, and i needed to make an exception for the one where we do what the timeout to run. :/